README.md

```diff
@@ -31,24 +31,42 @@ Making this more efficient and performant is what [IntersectionObserver](https:/
 
 ## Install
 
-Get the library using one of the following ways:
+This project uses [node](http://nodejs.org) and [npm](https://npmjs.com). Go check them out if you don't have them locally installed.
 
-1. **GitHub**
+```sh
+$ npm install --save lozad
+```
 
-Full build
+Alternatively you can use [Yarn](https://yarnpkg.com/en/).
 
- - [unminified] : https://raw.githubusercontent.com/ApoorvSaxena/lozad.js/master/dist/lozad.js
- - [minified] : https://raw.githubusercontent.com/ApoorvSaxena/lozad.js/master/dist/lozad.min.js
+```sh
+$ yarn add lozad
```

Comment:
place this above `npm` as `yarn` is recommended over `npm` owing to it's performance

Reply:
> place this above npm as yarn is recommended over npm owing to it's performance

Actually that's only personal preference. As `npm` is the default package manager of nodejs, it is more likely that the users have it installed.